src/Commands/AppConfig/Account/AccountListCommand.cs

```diff
@@ -23,7 +23,7 @@ List all App Configuration stores in a subscription. This command retrieves and
         stores available in the specified subscription. Results include store names returned as a JSON array.
         """;
 
-    [McpServerTool(Destructive = false, ReadOnly = true)]
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = "List App Configuration Stores")]
```

Comment:
These are all stored in the ArgumentDefinitions.cs file already.  Can we ref here instead of dupe?

Reply:
Meaning you already have the Titles? I didn't see them, but if that's the case, feel free to just close this.  Or if you mean something else, can you clarify?

Reply:
I was wrong.  We have arg text there, but not tool titles.  We have GetCommandName and GetCommandDescription,  

Could we use a GetCommandTitle() in the McpServerTool list?

Reply:
I'm thinking we have _commandTitle and GetCommandTitle in the BaseCommand so each command can expose Title in other ways, not just McpServerTool - we might want to expose via azmcp as well.